tests/conftest.py

```diff
@@ -16,6 +16,18 @@ def pytest_addoption(parser):
     group._addoption('--headless',
                      action='store_true',
                      help='enable headless mode for chrome.')
+    parser.addoption('--runslow', action='store_true',
```

Comment:
We can talk about this one. I'm wondering if maybe --runlong would be better. This test isn't really slow it just takes a long time to run because its a lot of tests for the same thing... we can always switch it out later if we write a test that is super slow. It's a little nitpicky.

Reply:
Good point. But I'm leaning towards sticking with `--runslow` for a couple reasons:

1. It's more consistent with the syntax of the decorator (`@pytest.mark.slow`) as well as the pytest documentation. (Unless there's another decorator for `mark.long`? I don't think it would make sense to code one from scratch.)
1. Eventually we will probably need to apply `@pytest.mark.slow` to future tests that run _slow_, but not necessarily _long_. If we are applying the same decorator to both slow and long tests, wouldn't it be repeating ourselves to define 2 different options?

Reply:
We can define our our marks for the decorator `pytest.mark.long`.  I think it makes more sense to keep going with slow according to your suggestions. Let's keep it that way. In effect, it is slowing down the throughput of the entire testing process. 👍